runner/src/main.rs

```diff
@@ -981,6 +981,14 @@ fn run_bazel_build() -> Step {
 fn run_bazel_test() -> Step {
     Step::Single {
         name: "bazel test".to_string(),
-        command: Cmd::new("bazel", &["test", "--", "//oak/...:all"]),
+        command: Cmd::new(
+            "bazel",
+            &[
+                "test",
+                "--",
+                "//oak/...:all",
+                "//remote_attestation/java/tests/...:all",
```

Comment:
We currently don't have Java tests in `oak_functions` since it would require having a Java Remote Attestation server.
We rely only on integration tests in form of examples.

Reply:
Thanks for checking :)